lib/rubyblok/helpers/storyblok_helper.rb

```diff
@@ -37,6 +37,29 @@ def rubyblok_blocks_tag(bloks)
     render_inline_partial template:, locals: { bloks: }
   end
 
+  def rubyblok_table_tag(blok)
```

Comment:
The solution was tested and it's working fine although I have mixed feelings about the solution.

I don't like the idea of having Ruby code mixed with HTML inside this class. I'm not sure if this is possible but it would be great to have something like the following instead:

```ruby
def rubyblok_table_tag(blok)
  RailsErbRenderer.render("partials/table.html.erb", blok:) # HTML with data returned
end
```

And your `partials/table.html.erb` file would have the template variable content.

Also, another solution would be to provide a table partial (via documentation or generators) with the same behavior.

I'd like to know what @ziklodi thinks about it.